src/sentry/tasks/integrations/slack/post_message.py

```diff
@@ -20,29 +23,27 @@ def _send_message_to_slack_channel(
     payload: Mapping[str, Any],
     log_error_message: str,
     log_params: Mapping[str, Any],
-    has_sdk_flag: bool | None = False,
+    has_sdk_flag: bool | None = True,  # TODO: remove all these
```

Comment:
Just curious. Why can't you remove it in this PR?

Reply:
this is in a task, it's possible if we remove it, we are still spawning tasks from one pod that pass the argument but the pod that's picking it up has updated code that doesn't have the argument in the function

Reply:
i had this issue where i added and was passing the argument to the task in one PR and it errored in Sentry for a couple minutes during deployment